packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTableSet.tsx

```diff
@@ -0,0 +1,292 @@
+                name: members[k].name,
+                typeTokens: members[k].typeTokens,
+                deprecated: members[k].deprecated,
+                defaultValue: members[k].defaultValue || ''
+              });
+            }
+
+            // the interface
+            if (propsName === jsonDocs.tables[j].name) {
+              iComponentProps.push({
+                propertyName: jsonDocs.tables[j].name,
+                description: jsonDocs.tables[j].descriptionHtml,
+                extendsTokens: jsonDocs.tables[j].extendsTokens,
+                title: jsonDocs.tables[j].kind ? jsonDocs.tables[j].name + ' ' + jsonDocs.tables[j].kind : jsonDocs.tables[j].name,
+                propertyType: PropertyType.interface,
+                property: interfaceMembers
+              });
+            } else {
+              preResults.push({
+                propertyName: jsonDocs.tables[j].name,
+                description: jsonDocs.tables[j].descriptionHtml,
+                extendsTokens: jsonDocs.tables[j].extendsTokens,
+                title: jsonDocs.tables[j].kind ? jsonDocs.tables[j].name + ' ' + jsonDocs.tables[j].kind : jsonDocs.tables[j].name,
+                propertyType: PropertyType.interface,
+                property: interfaceMembers
+              });
+            }
+
+            break;
+          }
+          case 'class': {
+            // class members are a mix of IApiInterfaceProperty and IMethod
+            const classMembers: IApiInterfaceProperty[] = [];
+            const classMethods: IMethod[] = [];
+
+            const members: ITableRowJson[] = jsonDocs.tables[j].members as ITableRowJson[];
+            for (let k = 0; k < members.length; k++) {
+              if (members[k].kind === 'Method') {
+                classMethods.push({
+                  description: members[k].descriptionHtml,
+                  name: members[k].name,
+                  typeTokens: members[k].typeTokens
+                });
+              } else {
+                classMembers.push({
+                  description: members[k].descriptionHtml,
+                  name: members[k].name,
+                  typeTokens: members[k].typeTokens,
+                  deprecated: members[k].deprecated,
+                  defaultValue: members[k].defaultValue || ''
+                });
+              }
+            }
+
+            // the class
+            preResults.push({
+              propertyName: jsonDocs.tables[j].name,
+              description: jsonDocs.tables[j].descriptionHtml,
+              extendsTokens: jsonDocs.tables[j].extendsTokens,
+              title: jsonDocs.tables[j].kind ? jsonDocs.tables[j].name + ' ' + jsonDocs.tables[j].kind : jsonDocs.tables[j].name,
+              propertyType: PropertyType.class,
+              property: classMembers,
+              methods: classMethods
+            });
+            break;
+          }
+        }
+      }
+    }
+
+    results = iComponentProps;
```

Comment:
You could probably skip declaring `results` above and just do `return [...iComponentProps, ...preResults]`